lib/base/config_file.c

```diff
@@ -521,12 +521,14 @@ heim_config_parse_dir_multi(heim_context context,
             return heim_enomem(context);
         }
         ret = heim_config_parse_file_multi(context, path, res);
+	/* Ignore malformed config files so we don't lock out admins, etc... */
+	if (ret && ret != ENOMEM)
+	    heim_warnx(context, "Ignoring '%s'", path);
```

Comment:
Sorry, this should have been 

`heim_warnx(context, ret, "Ignoring '%s'', path);